docs/notebooks/LORIS-API_Part1-HTTP.ipynb

```diff
@@ -0,0 +1,248 @@
+    "#### Headers\n",
+    "\n",
+    "\n",
+    "  - **Content-Length** *When the response contains a body (even for HEAD)*\n",
+    "  - **Content-Type** *The type of data returned*  \n",
+    "  - **Location** After a 303 response, the URL to GET the ressource\n",
+    "  - ...  \n",
+    "  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Quick example using curl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "curl -v -s --head https://demo.loris.ca/main.php 2>&1 |grep '[<>]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### With a html parser user agent (browser)\n",
+    "https://demo.loris.ca"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What if I do not want to use a mouse?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "curl -k -i -s \\\n",
+    "  -H 'Content-Type: application/x-www-form-urlencoded' \\\n",
+    "  -d 'username=demo&password=demo&login=Click+to+enter' \\\n",
+    "  https://demo.loris.ca/main.php"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## LORIS API ressources (endpoints)\n",
+    "\n",
+    "Our API documentation: https://github.com/aces/Loris/blob/minor/docs/API/LorisRESTAPI.md"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "curl -s https://demo.loris.ca/api/v0.0.3/login -d '{\"username\":\"\", \"password\": \"\"}'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Great, we have a token... now what?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Query the /candidates endpoint\n",
+    "# https://github.com/aces/Loris/blob/minor/docs/API/LorisRESTAPI.md#30-candidate-api\n",
+    "\n",
+    "token=''\n",
+    "curl -k -s \\\n",
+    "  -H \"Authorization: Bearer $token\" \\\n",
+    "  https://demo.loris.ca/api/v0.0.3/candidates/ "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Query the /candidate_list module\n",
+    "\n",
+    "token=''\n",
+    "curl -k -s \\\n",
+    "  -H \"Authorization: Bearer $token\" \\\n",
+    "  'https://demo.loris.ca/candidate_list/?format=json'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Scripting and parsing json \n",
+    "  \n",
+    "see: [part2 - Python script](https://github.com/aces/Loris/blob/2020-07-20-jupyterApiPart2/docs/notebooks/LORIS-API_Part2-Python-script.ipynb)"
```

Comment:
broken link

Reply:
I updated this to a link that will not be broken when the notebooks are merged (see my comment below)